cgmes/cgmes-extensions/src/main/java/com/powsybl/cgmes/extensions/CgmesIidmMapping.java

```diff
@@ -28,15 +28,31 @@ default String getName() {
 
     String getTopologicalNode(String equipmentId, int side);
 
-    boolean isMapped(String busId);
+    boolean isTopologicalNodeMapped(String busId);
```

Comment:
Can you add in the javadoc of this extension that it is a beta version and will probably evolve?

Reply:
This class already has a Warning with the following message:
"WARNING: this class is still in a beta version, it will change in the future"